apps/src/templates/studioHomepages/CourseBlocksTeacherGradeBands.jsx

```diff
@@ -1,28 +1,30 @@
 import React, {Component, PropTypes} from 'react';
 import ContentContainer from '../ContentContainer';
 import CourseBlocksGradeBands from './CourseBlocksGradeBands';
+import Responsive from '../../responsive';
 import i18n from "@cdo/locale";
 
 class CourseBlocksTeacherGradeBands extends Component {
   static propTypes = {
-    isRtl: PropTypes.bool.isRequired
+    isRtl: PropTypes.bool.isRequired,
+    responsive: PropTypes.instanceOf(Responsive).isRequired
   };
 
   cards = [
     {
-      heading: i18n.courseBlocksGradeBandsElementary(),
-      description: i18n.courseBlocksGradeBandsElementaryDescription(),
-      path: '/educate/curriculum/elementary-school'
+      heading: i18n.courseBlocksGradeBandsK5(),
+      description: i18n.courseBlocksGradeBandsK5Description(),
+      path: '/student/elementary'
```

Comment:
purposeful link changes?

Reply:
Good question.  I thought they'd been fixed by a previous PR but at any rate we want these links.